donkeycar/parts/pins.py

```diff
@@ -582,10 +582,7 @@ def set_duty_cycle(self, channel: int, duty_cycle: float):
         else:
             # duty cycle is fraction of the 12 bits
             pulse = int(4096 * duty_cycle)
-            try:
-                self.pwm.set_pwm(channel, 0, pulse)
-            except:
-                self.pwm.set_pwm(channel, 0, pulse)
+            self.pwm.set_pwm(channel, 0, pulse)
```

Comment:
I think this we should keep the try/catch and incorporate proper exception handling like: 
```python
try:
    self.pwm.set_pwm(channel, 0, pulse)
except Exception as e:
    logger.error(f'Error in talking to PCA9685: {str(e)}')
```

Reply:
Done.  Thanks.